src/shared/services/TopicService.js

```diff
@@ -56,7 +58,39 @@ export default class TopicService {
             ...topic,
             author: author.strip(),
             trainers: [],
-            users: [],
+            members: [],
+            id: newTopicKey
+        });
+    }
+
+    static async attend(topic, member) {
+        //const {trainers, users, author} = topic;
+        //check if user is not a member already
+
+        //check if user is not a trainer already
+
+        //check if a user is not an author
+        //update
+        return new Promise((resolve, reject) => {
+            db.ref(`topics/${topic.id}`).set({
+                ...topic,
+                members: [...topic.members, member]
+            }, function (err) {
```

Comment:
no arrow functions?

Reply:
fixed as well in https://github.com/warsawjs/topics-manager/pull/27/commits/b0de324e0b6e96429f62828ebc73fa58264f28ae